### DISCUSSION TITLE
OpenSSL update assessment, and Node.js project plans - what about NodeJS 16.X

### DISCUSSION THREAD

**Discussion by afaure-bkk** [nodejs/help#4936] [Q&A] [answered]
Hello,

We are assessing the impact of [Open SSL recent vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2022-1292) on our stack.

We have come across the note here :
https://nodejs.org/en/blog/vulnerability/openssl-fixes-in-regular-releases-may2022/

However, it does not specify the case for 16.X LTS releases.

Would you please advise on this point ?

Regards

**Comment by RafaelGSS** [comment, marked as answer]
v16.x was not affected by this vulnerability.
